prov/rxm/src/rxm_domain.c

```diff
@@ -37,13 +37,48 @@
 #include <fi_util.h>
 #include "rxm.h"
 
+int rxm_cntr_open(struct fid_domain *domain, struct fi_cntr_attr *attr,
+		  struct fid_cntr **cntr_fid, void *context)
+{
+	int ret;
+	struct util_cntr *cntr;
+	struct fi_cntr_attr cntr_attr = *attr;
+
+	cntr = calloc(1, sizeof(*cntr));
+	if (!cntr)
+		return -FI_ENOMEM;
+
+	switch (cntr_attr.wait_obj) {
+	case FI_WAIT_NONE:
+		break;
+	case FI_WAIT_UNSPEC:
+		cntr_attr.wait_obj = FI_WAIT_NONE;
+		break;
+	default:
+		ret = -FI_ENOSYS;
+		goto free;
+	}
```

Comment:
I think the util_cntr implementation can handle the wait objects.

Reply:
Yes, but cntr_wait routine should be fixed to ensure progressing in case of infinite timeout